plasmapy/physics/distribution.py

```diff
@@ -0,0 +1,85 @@
+"""Functions to deal with distribution : generate, fit, calculate"""
+
+
+from astropy import units
+
+from astropy.units import (UnitConversionError, UnitsError, quantity_input,
+                           Quantity)
+
+from ..constants import (m_p, m_e, c, mu0, k_B, e, eps0, pi, e)
+from ..atomic import (ion_mass, charge_state)
+
+import numpy as np
+
+from ..utils import _check_quantity, check_relativistic, check_quantity
+
+
+def Maxwellian_1D(v,T,part ="e", V_drift = 0* units.m/units.s):
+    r"""Return the probability at the velocity `v` in m/s to find a particle `part` in a plasma of temperature `T` following the Maxwellian distribution function
+
+    Parameters
+    ----------
+    v: Quantity
+        The velocity in units convertible to m/s
+
+    T: Quantity
+        The temperature in Kelvin
+
+    part: string, optional
+        Representation of the ion species (e.g., 'p' for protons, 'D+'
```

Comment:
Could we rename this to `particle`, and say `Representation of the particle species`?

Reply:
Ok for the argument name !